test/util/DbConnectionTestUtil.go

```diff
@@ -123,15 +129,32 @@ func (r *DbConnectionDbUtil) InsertUser() bool {
 
 func (r *DbConnectionDbUtil) connect() bool {
 
+	var nodeList v1.NodeList
```

Comment:
Add a comment telling why this is necessary.
Also, why do you assume only one node must be available.

Reply:
connecting to any node would work. So I'm taking a first one in the list.